### PR TITLE
Updated image in blog 2023-11-16

### PR DESCRIPTION
**Pull request by AmarNathChary**
updated image width in blog to user-friendly.

### CODE REVIEW

**Comment by AmarNathChary** [comment]
fixes : #44057

File: content/en/blog/_posts/2023-11-16-the-case-for-kubernetes-limits/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Firstly, let’s discuss bin-packing and cluster resource allocation. There’s
 
 When configuring fixed-fraction headroom limits, a proportional amount of this will be available to the pods. If the percentage of unallocated resources in the cluster is lower than the constant we use for setting fixed-fraction headroom limits (see the figure, line 2), all the pods together are able to theoretically use up all the node’s resources; otherwise there are some resources that will inevitably be wasted (see the figure, line 1). In order to eliminate the inevitable resource waste, the percentage for fixed-fraction headroom limits should be configured so that it’s at least equal to the expected percentage of unallocated resources.
 
-{{<figure alt="Chart displaying various requests/limits configurations" width="40%" src="requests-limits-configurations.svg">}}
+{{<figure alt="Chart displaying various requests/limits configurations" width="100%" src="requests-limits-configurations.svg">}}
```
Comment (sftim) [inline content/en/blog/_posts/2023-11-16-the-case-for-kubernetes-limits/index.md:38]: Even better:
```suggestion
{{<figure alt="Chart displaying various requests/limits configurations" class="diagram-medium" src="requests-limits-configurations.svg">}}
```
There is also `diagram-large` if appropriate.

Comment (Arhell) [reply content/en/blog/_posts/2023-11-16-the-case-for-kubernetes-limits/index.md:38]: lgtm

**Review by kumarankit999 — COMMENTED** [review]
taking @sftim suggestions, otherwise it is good!

**Comment by AmarNathChary** [comment]
@sftim committed changes PTAL!

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for the fix! The image dimensions appear more optimal, and the improvement is especially noticeable on smaller mobile screens compared to the previous version. 

/lgtm

[Preview Page](https://deploy-preview-44059--kubernetes-io-main-staging.netlify.app/blog/2023/11/16/the-case-for-kubernetes-resource-limits/#bin-packing-and-cluster-resource-allocation)

**Review by sftim — COMMENTED** [review]
Thanks

/approve

**Pull request merged**